app/models/patient.rb

```diff
@@ -49,10 +49,10 @@ class Patient < ApplicationRecord
             :initial_call_date,
             :line,
             presence: true
-  validates :primary_phone, format: /\d{10}/,
+  validates :primary_phone, format: /\A\d{10}\z/,
```

Comment:
Brakeman change - brakeman says 'keep it tight by anchoring start and end in your regexes'.